### PR TITLE
Wrote a blog on my experience as lfx-mentee

### PR DESCRIPTION
**Pull request by Debanitrkl**

### CODE REVIEW

File: blog/2021-06-20-lfx-mentorship-enriching-awschaos.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+image: /img/mentorship_experience.png
+tags: [Chaos Mesh, Chaos Engineering, LFX Mentorship, AWS Chaos]
+---
+
+![LFX Mentorship Experience](https://github.com/Debanitrkl/website/blob/master/static/img/mentorship_blog.jpeg)
+
+## About Me
+
+[I’m](https://mentorship.lfx.linuxfoundation.org/mentee/6a0bf7de-9e18-4acb-9a66-f5fecdbeb42e) a junior undergraduate majoring in Biomedical Engineering in the Department of Biotechnology and Medical Engineering at the [National Institute of Technology Rourkela](https://nitrkl.ac.in/), India. For someone who started to code only because I was fascinated by it, it was all a journey of self-learning, filled with various adversities. But when I started with open-source contributions, it was all very beginner-friendly and I came across a lot of people who helped me learn the tech stack better.
+
```
Comment (milasuperstar) [inline blog/2021-06-20-lfx-mentorship-enriching-awschaos.md:17]:
```suggestion
<!--truncate-->
```

File: blog/2021-06-20-lfx-mentorship-enriching-awschaos.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+Afterwards, I applied for the CFP for [Kubernetes Community Days Bangalore](https://community.cncf.io/events/details/cncf-kcd-bengaluru-presents-kubernetes-community-days-bengaluru/), scheduled virtually from June 25–26, 2021, and was selected as a speaker and now I’m all set to present my talk there.
+
+## Graduation and Next steps
+
+Yayyyy!! So finally I graduated from the program, thanks to my mentor [Zhou Zhiqiang](https://mentorship.lfx.linuxfoundation.org/mentor/e78b3177-160c-4566-9f3d-8fc9b2ec3cea), without whom, this wouldn’t have been possible.
+
+I had an amazing time with the Chaos Mesh community, with the amazing members supporting and helping me throughout the journey. I look forward to contributing more to this project and being more active in the community.
+
+## Join us
```
Comment (milasuperstar) [inline blog/2021-06-20-lfx-mentorship-enriching-awschaos.md:66]:
```suggestion
## Join the Chaos Mesh community
```

File: blog/2021-06-20-lfx-mentorship-enriching-awschaos.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+Initially, I was asked to implement one type of AWS Chaos as part of this project, but as I started exploring more about it, I found [awsssmchaosrunner](https://github.com/amzn/awsssmchaosrunner), and given its functionality, we wanted to integrate it into Chaos Mesh.
+
+We planned to do it in two parts, one part is the “[runner thing](https://github.com/STRRL/awsssmchaosrunner-cli)” project, which integrates with awsssmchaosrunner, that part should be written in kotlin, and a docker image is to be built out of it.
+
+Another part is the definition of the AWS Chaos and its [controller](https://github.com/chaos-mesh/chaos-mesh/pull/1919), which is to be written in go, the controller of AWS Chaos will create a pod with that “kotlin cli image”, and send commands to AWS.
+
+## Other opportunities
+
+I was invited to one of the [community meetings](https://www.youtube.com/watch?v=ElG0pHRoXwI&t=2s) of Chaos Mesh where I showcased my project.
```
Comment (milasuperstar) [inline blog/2021-06-20-lfx-mentorship-enriching-awschaos.md:56]:
```suggestion
I was invited to one of the Chaos Mesh [community meetings](https://www.youtube.com/watch?v=ElG0pHRoXwI&t=2s) towards the end of the mentorship where I showcased my project.
```

File: blog/2021-06-20-lfx-mentorship-enriching-awschaos.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+Afterwards, I applied for the CFP for [Kubernetes Community Days Bangalore](https://community.cncf.io/events/details/cncf-kcd-bengaluru-presents-kubernetes-community-days-bengaluru/), scheduled virtually from June 25–26, 2021, and was selected as a speaker and now I’m all set to present my talk there.
+
+## Graduation and Next steps
+
+Yayyyy!! So finally I graduated from the program, thanks to my mentor [Zhou Zhiqiang](https://mentorship.lfx.linuxfoundation.org/mentor/e78b3177-160c-4566-9f3d-8fc9b2ec3cea), without whom, this wouldn’t have been possible.
```
Comment (milasuperstar) [inline blog/2021-06-20-lfx-mentorship-enriching-awschaos.md:62]:
```suggestion
Yayyyy!! After 12 weeks, I graduated from the program, thanks to my mentor [Zhou Zhiqiang](https://mentorship.lfx.linuxfoundation.org/mentor/e78b3177-160c-4566-9f3d-8fc9b2ec3cea), without whom, this wouldn’t have been possible.
```

**Comment by g1eny0ung** [comment]
@Debanitrkl There are some unexpected changes. Please rebase your commits and force push. Only blog-related content is needed.

**Comment by Debanitrkl** [comment]
> @Debanitrkl There are some unexpected changes. Please rebase your commits and force push. Only blog-related content is needed.

Updated the PR do check

File: docs/development_guides/setup_env.md [hunk 5]
```diff
@@ -29,7 +29,7 @@ Make sure you have the above prerequisites met. Now follow the steps below to pr
 2. Install the Kubernetes API development framework - [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) and [kustomize](https://github.com/kubernetes-sigs/kustomize).
 
    ```bash
-   make ensure-all
+   make all
```
Comment (g1eny0ung) [inline docs/development_guides/setup_env.md:32]: I don't think it needs to be changed here.

Comment (Debanitrkl) [reply docs/development_guides/setup_env.md:32]: Yeah actually that was a change that I tried doing months ago but then closed the PR, don't know why it's appearing again here

Comment (g1eny0ung) [reply docs/development_guides/setup_env.md]: Ok, I've already resolved this problem. Next, I will merge this PR.

**Review by g1eny0ung — APPROVED** [review]

**Pull request merged**